polaris-react/src/components/DropZone/DropZone.stories.tsx

```diff
@@ -378,14 +378,36 @@ export function SmallSized() {
       <BlockStack gap="200">
         <div>
           <Text as="h2" variant="headingMd">
-            Small sized Drop zone
+            Small sized Drop zone (50px)
```

Comment:
@sam-b-rose , should it be 
```suggestion
            Small sized Drop zone (40px)
```
?

Reply:
I wanted to test that this would work well with any parent element that was sized to 50x50px. We could remove this story though and just use the single 40x40 test example 👍

Reply:
I'm not sure I should remove this story, as it was here before, 🤔 . Or do you mean remove part with (40px)?
I just added new one `Small sized Drop zone without outline`

Reply:
oh, now I see, you added this new story with your commit. Agree, removing it

Reply:
Thank you!